infrastructure/cdn-in-a-box/traffic_ops/Dockerfile

```diff
@@ -105,5 +105,5 @@ ADD infrastructure/cdn-in-a-box/enroller/server_template.json \
 
 ADD infrastructure/cdn-in-a-box/traffic_ops_data /traffic_ops_data
 
-RUN chown -R trafops:trafops /opt/traffic_ops
+# RUN chown -R trafops:trafops /opt/traffic_ops
```

Comment:
?

Reply:
I'm honestly tempted to just remove that line it doesn't serve any apparent purpose, and takes _forever_ because it's recursing over probably tens of thousands of files. I had to disable it to get the builds to run fast enough to debug that earlier problem in one night..

I guess I shouldn't in this PR though.

Reply:
I would be fine leaving it in this PR if it doesn’t cause issues which sounds like it doesn’t